src/main/java/org/jenkinsci/plugins/docker/workflow/client/WindowsLinuxDockerClient.java

```diff
@@ -0,0 +1,67 @@
+package org.jenkinsci.plugins.docker.workflow.client;
+
+import com.google.common.base.Optional;
+import hudson.EnvVars;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Node;
+import hudson.util.ArgumentListBuilder;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.io.*;
+import java.nio.charset.Charset;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class WindowsLinuxDockerClient extends WindowsDockerClient {
```

Comment:
Taking the code, it would be rather a Unix/POSIX container launcher. Not that important tho